src/Plugin/DataType/StrawberryValuesViaJmesPathFromJson.php

```diff
@@ -52,15 +54,24 @@ public function process($langcode = NULL)
       return;
     }
     $item = $this->getParent();
-    if (!empty($item->value)) {
-      /* @var $item StrawberryFieldItem */
+    if (!empty($item->getValue())) {
       $definition = $this->getDataDefinition();
       // This key is passed by the property definition in the field class
       // jsonkey in this context is a string containing one or more
       // jmespath's separated by comma.
-      $jmespaths = $definition['settings']['jsonkey'];
+      $jmespaths = $definition['settings']['jsonkey'] ?? '';
       $is_date = $definition['settings']['is_date'] ?? FALSE;
-      $jmespath_array = array_map('trim', explode(',', $jmespaths));
+      $is_date_range = $definition['settings']['is_date_range'] ?? FALSE;
+      $pattern = '/[,]+(?![^\[]*\]|[^\(]*\)|[^\{]*\})/';
```

Comment:
Confirmed this regex pattern works for my jmespath expressions containing commas.